platform-sdk/swirlds-config-extensions/src/main/java/com/swirlds/config/extensions/sources/SimpleConfigSource.java

```diff
@@ -289,16 +307,50 @@ public void reset() {
     /**
      * {@inheritDoc}
      */
+    @NonNull
+    @Override
+    public Set<String> getPropertyNames() {
```

Comment:
I assume as a next step we can refactor the AbstractConfigSource and find a good pattern to support lists in an abstract way.

Reply:
https://github.com/hashgraph/hedera-services/issues/17102